grails-app/controllers/au/org/emii/aatams/SensorController.groovy

```diff
@@ -32,8 +32,6 @@ class SensorController extends ReportController {
         def tag = tagFactoryService.lookupOrCreate(params.tag)
         assert(tag)
 
-        params.tag = tag
```

Comment:
> Perhaps we could get @xhoenner to test it on the test or dev instances - paying particular attention to cross-referencing the id's between the tables?

Yes happy to test that thoroughly whenever it's ready. Keep in mind that I've got tomorrow off though.